test/functional/commands/keyboard-e2e-specs.js

```diff
@@ -42,6 +44,12 @@ async function runTextEditTest (driver, testText, keys = false) {
   return el;
 }
 
+async function clearKeyEvents (driver) {
```

Comment:
sorry I'm a little confused what this function name means. How does this function clear keyevents? Or is it sending "clear key" events? Maybe a note about what button this is tapping and how that is related might help?